### ISSUE TITLE
Dailymotion error: unable to parse JSON

### ISSUE THREAD

**Issue by eme00** [chrippa/livestreamer#666]
hi guys i'm really noob on this thing so pls help me. i'm trying to stream dailymotion video but i'm keep on getting error saying like this 
error: unable to parse JSON: No JSON object could be decoded <u'<!DOCTYPE html>\n<html>\n\t<head> ...>

Youtube video is working fine but cannot stream dailymotion video :(

i tried to stream by using this both  command 

livestreamer dailymotion.com/video/x2d774c_happy-new-year-2014-webhd-rip-x264-aac-ddr-
00h00m00s-00h59m49s-002_shortfilms best

livestreamer https://www.dailymotion.com/video/x2d774c_happy-new-year-2014-webhd-rip-x264-aac-ddr-00h00m00s-00h59m49s-002_shortfilms

but still getting this error any fix for this pls Help me!!

**Comment by walker9** [comment]
Hi,

I have the same Problem detect with Dailymotion Videos.

--version 1.12.2 on Windows


Got this Error:

error: Unable to parse JSON: No JSON object could be decoded (u'#EXTM3U\n#EXT-X-STREAM-INF:PROGRA ...)

No Conflicts with Proxy or Firewall i guess, cause the Gaming Streams from Dailymotion (http://games.dailymotion.com/) are possible to open.
![image2](https://cloud.githubusercontent.com/assets/15380148/10822224/8c156754-7e56-11e5-97fc-664e29fbbeaf.png)

**Comment by 0880** [comment]
> error: Unable to parse JSON: No JSON object could be decoded (u'#EXTM3U\n#EXT-X-STREAM-INF:PROGRA ...)

I can confirm that.

````
livestreamer "http://www.dailymotion.com/video/x3bdcvj" best
[cli][info] Found matching plugin dailymotion for URL http://www.dailymotion.com/video/x3bdcvj
error: Unable to parse JSON: No JSON object could be decoded (u'#EXTM3U\n#EXT-X-STREAM-INF:PROGRA ...)
````

The [JSON] (http://www.dailymotion.com/sequence/full/x3bdcvj) file is parsed correctly, but the [M3U8] (http://www.dailymotion.com/cdn/manifest/video/x3bdcvj.m3u8?auth=1446361359-2562-y2odlrw4-9c5d7293f09ba53f8a75f9d0ddc64997) file cannot be processed.

**Comment by Shuunen** [comment]
I have the same ;'(

**Comment by ghost** [comment]
@Shuunen 

https://github.com/chrippa/livestreamer/commit/dc96641de963186c589b76f7a00a2da449cba160

```
livestreamer "http://www.dailymotion.com/video/x3bdcvj" best
[cli][info] Found matching plugin dailymotion for URL http://www.dailymotion.com/video/x3bdcvj
[cli][info] Available streams: 184p (worst), 288p, 480p, 720p, 1080p (best)
[cli][info] Opening stream: 1080p (hls)
```

**Comment by Shuunen** [comment]
@0880 : thanks :) so it's into develop right ? we should wait for the next release of livestreamer ? Because releases seems to be freezed since 1.12.2 from May 2015 :s

**Comment by ghost** [comment]
> we should wait for the next release of livestreamer ?

Not necessarily. On Windows you can use the [nightly build](http://livestreamer-builds.s3.amazonaws.com/livestreamer-latest-win32.zip).

**Comment by Shuunen** [comment]
Oh thanks mate :) works like a charm :+1:

**Comment by walker9** [comment]
With the Nightly Build it also works for me now! thx :)
